packages/@okta/vuepress-site/docs/guides/implement-client-creds/use-flow/index.md

```diff
@@ -32,4 +34,52 @@ If the credentials are valid, the application receives back an access token:
 }
 ```
 
+### Base64 encode the client ID and client secret
+
+Use this section to Base64 encode the client ID and secret. When you finish encoding, you can then use the encoded client ID and secret in the HTTP Authorization header in the following format: `'authorization: Basic <Base64 encoded client ID and secret>'`
+
+1. From the Developer Console, click **Applications**, select the application that you want to use, and then on the **General** tab, copy the client ID and secret.
+
+2. Launch your preferred text editor and then paste the client ID and secret into a new file.
+
+3. Place the client ID and secret on the same line and insert a colon between them: `clientid:clientsecret`
+
+4. Copy the `clientid:clientsecret` line.
+
+5. Access the [base64encode.org](https://www.base64encode.org/) web site and paste the `clientid:clientsecret` line in the **Encode to Base64 format** box.
+
+6. Leave **UTF-8** as the **Destination character set** and click **Encode**.
+
+7. Copy the encoded line that appears.
+
+
+To use the command line for Mac and Linux:
+
+1. Follow steps one through four above.
```

Comment:
As a matter of style, should we use numerals for one and four? I.e., "1. Follow steps 1 through 4 above."  It might look off as it's part of a numbered point. I'm fine either way. (Applies to Windows section below as well).

Reply:
thanks, yes - Microsoft Manual of Style says steps are an exception to the "spell out numbers below 10" rule. I'll fix!